build/docker/services/base/Dockerfile

```
@@ -15,6 +15,11 @@ LABEL org.label-schema.name="htcondor/base:${VERSION}-el${EL}${SUFFIX}" \
       org.label-schema.vendor="HTCondor" \
       org.label-schema.license="Apache-2.0"
 
+# Ensure that the 'condor' UID/GID matches across containers
+RUN groupadd -g 64 -r condor
+RUN useradd -r -g condor -d /var/lib/condor -s /sbin/nologin \
+    -u 64 -c "Owner of HTCondor Daemons" condor
```

No point in separate layers for the groupadd and the useradd.
```suggestion
RUN groupadd -g 64 -r condor && \
    useradd -r -g condor -d /var/lib/condor -s /sbin/nologin \
      -u 64 -c "Owner of HTCondor Daemons" condor
```